operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -2125,21 +2125,17 @@ message KialiServiceConfig {
 message KialiDashboardConfig {
   string secretName = 1;
 
-  string usernameKey = 2;
+  google.protobuf.BoolValue viewOnlyMode = 2;
```

Comment:
well - I guess the ship has sailed on the protobuf for KialiDashBoardConfig. Make sure to keep compatibility with the old protobuf.

Reply:
This "usernameKey" (and a couple others in here - "passwordKey" is another) hasn't been used for a long time by Kiali and hasn't been needed by several Istio releases at least. This is cruft that should have been removed a long time ago. I know these haven't been used at all by Kiali for a long time. I didn't think we need it here in this .proto since it isn't used in the Kiali values.yaml (in fact, it has been removed from Kiali's values.yaml since at least Istio 1.1).

Reply:
Sounds fair. I think we need to consider for a 1.7 work item where the Istio installation and operations API heads next. For 1.6, I think we should be considerate of the amount of change we are pushing on our operators.